### PR TITLE
Refine StaticRnn

### PR DESCRIPTION
**Pull request by chengduoZH**
image

......

image
image
image

### CODE REVIEW

**Comment by phlrain** [comment]
LGTM

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -267,8 +267,43 @@ class StaticRNN(object):
...
+        >>>    hidden = fluid.layers.fc(input=[word, prev], size=hidden_size, act='relu')
+        >>>    rnn.update_memory(prev, hidden)  # set prev to hidden
+        >>>    rnn.step_output(hidden)
+        >>>
+        >>> result = rnn()
+
+    The StaticRNN will unfold sequence into time steps. Users need to define
+    how to process each time step during the :code:`with` step.
+
+    The :code:`memory` is used staging data cross time step. The initial value of
```
Comment (shanyi15) [inline python/paddle/fluid/layers/control_flow.py]: The :code:`memory` is used staging data cross time step

==
be used to do?

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -267,8 +267,43 @@ class StaticRNN(object):
...
+    memory can be zero or another variable.
```
Comment (shanyi15) [inline python/paddle/fluid/layers/control_flow.py]: or another

==
https://baijiahao.baidu.com/s?id=1621790191205802286&wfr=spider&for=pc

**Review by shanyi15 — APPROVED** [review]
LG comments

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**